Gemfile.lock

```diff
@@ -243,7 +243,7 @@ GEM
     attr_extras (7.1.0)
     awesome_print (1.9.2)
     aws-eventstream (1.3.0)
-    aws-partitions (1.976.0)
+    aws-partitions (1.977.0)
```

Comment:
Are the changes in this file intentional?

Reply:
➕

Reply:
Master was messed up with bundle audit. I will pull from master.